shared/js/dataDriver.js

```diff
@@ -284,6 +301,7 @@ loop.DataDriver = function() {
       }));
       this._sse.addEventListener("open", this);
```

Comment:
Yeah, like a close method:

```javascript
close() {
  if (this._sse) {
    this._sse.removeEventListener("open", this); // ... and other methods
  }
  this._sse = null;
}
```

Then we know there aren't shadow connections or event listeners out there that can cause confusion.